Getting-Started/Backoffice/index.md

```diff
@@ -80,3 +80,10 @@ A parameter editor defines the usage of a property editor for use as a parameter
 
 ### User
 A user is someone who has access to the **Umbraco backoffice** and is not to be confused with Member. When Umbraco has been installed a user will automatically be generated with the login (email) and password entered during installing. Users can be created, edited and managed in the User section.
+
+### Member vs User
+| Member | User |
+|:-------|:-----|
+| To view protected content on the frontend (such as public facing website), **Member** needs to be logged for granting access. | **Users** can access the admin backend, they can also be restricted to view specific content. |
```

Comment:
There's a few grammatical errors here I think it should be something like:

To view protected content on the frontend (such as on a public facing website), a **Member** needs to be logged in to be granted access. | **Users** can access the admin backend, they can also be restricted to view specific content.

Reply:
```suggestion
| To view protected content on the frontend (such as public facing website), **Member** needs to be logged for granting access. | **Users** can access the admin backend, they can also be restricted to view specific content. |
```
I can't change this and even other Conversations because of conflicts in  resolving the pull request .